github/projects.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// ProjectsService handles communication with the project V2
+// methods of the GitHub API.
+//
+// GitHub API docs: https://docs.github.com/rest/projects/projects
+type ProjectsService service
+
+func (p ProjectV2) String() string { return Stringify(p) }
+
+// ListProjectsOptions specifies optional parameters to list organization projects.
+type ListProjectsOptions struct {
+	// Q is an optional query string to filter/search projects (when supported).
+	Q string `url:"q,omitempty"`
+	ListOptions
+	ListCursorOptions
+}
+
+// ListOrganizationProjects lists Projects V2 for an organization.
+//
+// GitHub API docs: https://docs.github.com/rest/projects/projects#list-organization-projects
+//
+//meta:operation GET /orgs/{org}/projectsV2
+func (s *ProjectsService) ListOrganizationProjects(ctx context.Context, org string, opts *ListProjectsOptions) ([]*ProjectV2, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/projectsV2", org)
+	u, err := addOptions(u, opts)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	req.Header.Set("Accept", mediaTypeProjectsPreview)
```

Comment:
Do we need this header?

Reply:
It's recommended to set it https://docs.github.com/en/rest/projects/projects?apiVersion=2022-11-28#list-projects-for-organization